src/tr.py

```diff
@@ -40,12 +40,12 @@ def translateText(context, text, n=None):
         try:
```

Comment:
I rewritten it 2 years ago ): https://github.com/Bitmessage/PyBitmessage/pull/1389/files?file-filters%5B%5D=.py#diff-a804aa473dd72c3caf797608b9abc7931b439092f114a9e6b60e9a2aaf5fccc4

Reply:
Didn't know this existed. Do you suggest starting my work from that or incorporating some changes into my fork?

Reply:
You can see all the forks on the network page: https://github.com/Bitmessage/PyBitmessage/network I suggest you to keep trying if you wanna merge, but be prepared to long wait. Existing PR's and issues are also worth reading, for example #1712